open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -39,22 +54,20 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
  called .env in the same folder and amend the values as required.
  * Once the .env file is in place, build and start the server using `npm start`. Use other npm parameters as specified in [package.json](nodejs/package.json).  
 
```

Comment:
Where we explain the content of the environment variable it would be nicer to describe the "remoteSevrerName" as the name of the View Server and to describe the "remoteURL" as the serverRootURL of the View Server.